Makefile

```diff
@@ -12,7 +12,7 @@ all: image
 image:
 	@echo "Creating container"
 	@echo "- Building container"
-	- docker build -t phly-docker-php-swoole -f ./Dockerfile .
+	- docker build -t phly-docker-php-swoole .
```

Comment:
Reverting this, but this is because of a change that happened later: the Makefile now targets two different architectures.